package.json

```diff
@@ -7,10 +7,13 @@
     "build:watch": "lerna run build:watch --stream --no-sort --concurrency 9999",
     "clean": "lerna run --stream clean && lerna clean --yes",
     "fix": "lerna run --stream --concurrency 1 fix",
+    "fix-since": "lerna run --since --stream --concurrency 1 fix",
     "link:yarn": "lerna run --stream --concurrency 1 link:yarn",
-    "lint": "lerna run --stream --concurrency 1 lint",
+    "lint": "lerna run --stream --concurrency 9999 lint",
+    "lint-since": "lerna run --since --stream --concurrency 9999 lint",
     "prepare": "yarn build",
-    "test": "lerna run test --stream --concurrency 9999"
+    "test": "lerna run test --stream --concurrency 9999",
+    "test-since": "lerna run test --since --stream --concurrency 9999"
```

Comment:
If I'm reading the docs correctly, the `--since` flag here will effectively limit these commands to just changes since the latest release? If so, that's pretty cool, and it gives us a good incentive to keep releasing often

Reply:
yeah! added just because I saw it in the docs, i think it'll help us in the case where pkg A's tests are busted but you're only improving pkg B (hopefully we never have a state though where any package/test case is broken beyond repair though 🤞 )